### ISSUE TITLE
Global Feature Importance

### ISSUE THREAD

**Issue by Tejamr** [interpretml/interpret#340]
Hi, Is there any particular paper or tutorial which clarifies the mathematical calculation behind the scores if yes,please provide as I am unable to crack the source code .
Please provide the entire mathematical calculation behind the below dataset (Diabetes dataset , link is given below ).
I have applied EBM and unable to find how calculations happened in both Global and Local Importance .
Here are the figures regarding those .
Please provide notebook or any paperwork how each value is calculated in both the cases .(from scratch)

Note : Mathematical clarification part will be more beneficial if provided in a granule manner .
![newplot](https://user-images.githubusercontent.com/91931018/168212274-70b05b9e-554e-4df8-a983-00d046d7b25c.png)
![newplot (3)](https://user-images.githubusercontent.com/91931018/168212293-4c7a0219-6923-4fdc-8d76-c26d6b3f550e.png)

[diabetes.csv](https://github.com/interpretml/interpret/files/8684116/diabetes.csv)

**Comment by paulbkoch** [comment]
Here's a link to the original paper that started EBMs:
https://www.cs.cornell.edu/~yinlou/papers/lou-kdd12.pdf

The boosting process to generate EBMs is well covered in the paper.  Once an EBM is generated, evaluation is straightforward.  For each feature you discretize the feature based on the contents of the bins_ attribute of the EBM.  Using the bin index from this discretization, you lookup the score value for that feature in the term_scores_ attribute.  Sum these up for each feature, and also add in the intercept, and that's the prediction for a regression EBM.  For classification, the number that you just calculated is a logit, so pass the logit into softmax to calculate the probabilities.
